docs/acls.md

```diff
@@ -0,0 +1,428 @@
+3. ingress network policy, default deny priority = `1000`, allow priority = `1001`
+
+## Egress Firewall
+
+Egress Firewall creates 1 ACL for every specified rule, with `ExternalIDs["k8s.ovn.org/owner-type"]=EgressFirewall`
+e.g. given object:
+
+```yaml
+kind: EgressFirewall
+apiVersion: k8s.ovn.org/v1
+metadata:
+  name: default
+  namespace: default
+spec:
+  egress:
+    - type: Allow
+      to:
+        dnsName: www.openvswitch.org
+    - type: Allow
+      to:
+        cidrSelector: 1.2.3.0/24
+      ports:
+        - protocol: UDP
+          port: 55
+    - type: Deny
+      to:
+        cidrSelector: 0.0.0.0/0
+```
+
+will create 3 ACLs:
+
+```
+action              : allow
+direction           : to-lport
+external_ids        : {
+    "k8s.ovn.org/id"="default-network-controller:EgressFirewall:default:10000", 
+    "k8s.ovn.org/name"=default, 
+    "k8s.ovn.org/owner-controller"=default-network-controller, 
+    "k8s.ovn.org/owner-type"=EgressFirewall, 
+    priority="10000"
+}
+label               : 0
+log                 : false
+match               : "(ip4.dst == $a5272457940446250407) && ip4.src == $a4322231855293774466"
+meter               : acl-logging
+name                : "EF:default:10000"
+options             : {}
+priority            : 10000
+severity            : []
+
+action              : allow
+direction           : to-lport
+external_ids        : {
+    "k8s.ovn.org/id"="default-network-controller:EgressFirewall:default:9999", 
+    "k8s.ovn.org/name"=default, 
+    "k8s.ovn.org/owner-controller"=default-network-controller, 
+    "k8s.ovn.org/owner-type"=EgressFirewall, 
+    priority="9999"
+}
+label               : 0
+log                 : false
+match               : "(ip4.dst == 1.2.3.0/24) && ip4.src == $a4322231855293774466 && ((udp && ( udp.dst == 55 )))"
+meter               : acl-logging
+name                : "EF:default:9999"
+options             : {}
+priority            : 9999
+severity            : []
+
+action              : drop
+direction           : to-lport
+external_ids        : {
+    "k8s.ovn.org/id"="default-network-controller:EgressFirewall:default:9998", 
+    "k8s.ovn.org/name"=default, 
+    "k8s.ovn.org/owner-controller"=default-network-controller, 
+    "k8s.ovn.org/owner-type"=EgressFirewall, 
+    priority="9998"
+}
+label               : 0
+log                 : false
+match               : "(ip4.dst == 0.0.0.0/0 && ip4.dst != 10.244.0.0/16) && ip4.src == $a4322231855293774466"
+meter               : acl-logging
+name                : "EF:default:9998"
+options             : {}
+priority            : 9998
+severity            : []
+```
+
+Egress firewall should be applied after egress network policy independently, to make sure that connection that are
+allowed by network policy, but denied by egress firewall will be dropped.
+
+## Multicast
+
+Multicast creates 2 types of ACLs: global default ACLs with `ExternalIDs["k8s.ovn.org/owner-type"]=MulticastGlobal`
```

Comment:
I think a brief sentence or 2 on why we need global multicast ACLs would be helpful.

Reply:
can you add those sentences explaining what those ACLs are for?

Reply:
oh I just didn't see you modified it there.